src/tests/components/searchForm.test.js

```diff
@@ -1,12 +1,9 @@
 import React from "react";
 import { render } from "@testing-library/react";
 import SearchForm from "../../components/SearchForm";
-import setSearchText from "../../components/App";
-import handleCitySearch from "../../components/App";
-
 
 describe("Search Form", () => {
-    it("renders correctly", () => {
+  it("renders correctly", () => {
```

Comment:
None of these tests are passing, because you are passing in undefined values to the components you are trying to render. You need to use a valid props object like so:

```
const validProps = {
  searchText: "",
  setSearchText: jest.fn(),
  onSubmit: jest.fn(),
};
```

And pass its values into the components like so:
```
<SearchForm
  searchText={validProps.searchText}
  setSearchText={validProps.setSearchText}
  onSubmit={validProps.onSubmit}
/>
```